ai_edge_torch/generative/examples/stable_diffusion/README.md

```diff
@@ -2,13 +2,48 @@
 This example shows how to use the Edge Generative API to convert a PyTorch Stable Diffusion v1.5 model to TFLite model, and run the image generation inference.
 
 ## Convert PyTorch to TFLite model
+The example provides two source checkpoints mapping. One is original HuggingFace repo, and the other one is a third party PyTorch implementation of Stable Diffusion. Users need to specify the source checkpoints mapping format for conversion script.
```

Comment:
i guess it's better to only support safetensors for now..

Reply:
Done